lib/child-writer/writer-proxy.js

```diff
@@ -92,6 +92,7 @@ permissions.isElevated().then((elevated) => {
         ELECTRON_RUN_AS_NODE: 1,
         IPC_SERVER_ID: process.env.IPC_SERVER_ID,
         IPC_CLIENT_ID: process.env.IPC_CLIENT_ID,
+        ETCHER_RUNNING: 1,
```

Comment:
What is this variable for? What does it mean?

Reply:
Basically, we need some way to communicate to the writer-proxy that it was called. Previously it was just a check of whether it was running with privileges or not, but now it always does. Knowing this, it creates the IPC channel. We could use `SKIP` instead but I thought it'd be easier to understand with a new variable. I'll add a comment explaining this.

Reply:
So basically if you elevate the app with `gksudo`, then `is-elevated` still returns `false`?

Reply:
Oh no, the opposite, it always returns true which causes it to skip the step and no IPC channel is established which means no progress.

Reply:
@Shou have you added the comment explaining this yet? I'm not seeing it, or I'm missing something.